src/components/Calculator.test.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import Design from './Calculator';
+
```

Comment:
- [ ] Kindly as the requirements say `Use React Testing Library to simulate user interaction.` Kindly add some tests for user interactions with the calculator.

For example
```
test('Test Calculator', async () => {
  render(<Calculator />);
  fireEvent.click(screen.getByText('9'));
  const result = await waitFor(() => screen.getByTestId('result'));
  expect(result.textContent).toBe('9');
});
```